content/1.7/back-office/shop-parameters/search/tags/tags-listing.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Tags listing
+weight: 1
+---
+# Tags listing
+
+The tagging feature enables to associate products with keywords. Customers can use the keywords to easily and quickly find the products they are looking for.
+
+## Multistore
```

Comment:
This whole part is wrong, there is no shop association block while creating or editing a tag, so it's added for all shops and then the lists displays all the tags no matter the context